pkg/assembler/graphql/resolvers/isDependency.resolvers_test.go

```diff
@@ -120,3 +124,100 @@ func TestIngestDependencies(t *testing.T) {
 		})
 	}
 }
+
+func TestDependencyTypeIsValid(t *testing.T) {
+	tests := []struct {
+		Name       string
+		ExpIsValid bool
+	}{
+		{
+			Name:       "DIRECT",
+			ExpIsValid: true,
+		},
+		{
+			Name:       "INDIRECT",
+			ExpIsValid: true,
+		},
+		{
+			Name:       "UNKNOWN",
+			ExpIsValid: true,
+		},
+		{
+			Name:       "Some other value",
+			ExpIsValid: false,
+		},
+		{
+			Name:       "",
+			ExpIsValid: false,
+		},
+	}
+
+	for _, test := range tests {
+		if model.DependencyType(test.Name).IsValid() != test.ExpIsValid {
+			t.Errorf("Expected dependency type %s to have validity %v but it did not", test.Name, test.ExpIsValid)
+		}
+	}
+}
+
+func TestIngestDependenciesDependencyTypeValidity(t *testing.T) {
+	tests := []struct {
+		Name            string
+		DependencyTypes []string
+		ExpAllValid     bool
+	}{
+		{
+			"nil",
+			nil,
+			true,
```

Comment:
Go ahead and list the field names here. It can be easy to forget what is what when these get long
```go
			Name: "nil",
			DependencyTypes: nil,
			ExpAllValid: true,
```

Reply:
Fixed in a2703d24aea4b9b780970a798f5839956774ee71.